unittests/test_meta.py

```diff
@@ -201,9 +204,28 @@ def hook_b(self):
 
     assert not Bar.hook_in_stage('hook_a', 'post_setup')
     assert not Bar.hook_in_stage('hook_b', 'pre_compile')
-    assert Bar.hook_in_stage('hook_c', 'post_run_wait')
+    assert Bar.hook_in_stage('hook_c', 'post_run')
     assert Bar.hook_in_stage('hook_a', 'pre_sanity')
 
+    class Baz(MyMeta):
```

Comment:
Can you add a comment here that you are testing attaching hooks to a random class?